README.md

```diff
@@ -1,36 +1,36 @@
 [![VFX Platform](https://img.shields.io/badge/vfxplatform-2020-blue.svg)](http://www.vfxplatform.com/)
-[![Python 2.6 2.7 3.7](https://img.shields.io/badge/python-2.6%20%7C%202.7%20%7C%203.7-blue.svg)](https://www.python.org/)
-[![Reference Documentation](http://img.shields.io/badge/doc-reference-blue.svg)](http://developer.shotgunsoftware.com/python-api)
+[![Python 2.7 3.7](https://img.shields.io/badge/python-2.7%20%7C%203.7-blue.svg)](https://www.python.org/)
+[![Reference Documentation](http://img.shields.io/badge/doc-reference-blue.svg)](http://developer.shotgridsoftware.com/python-api)
 [![Build Status Linux](https://secure.travis-ci.org/shotgunsoftware/python-api.svg?branch=master)](http://travis-ci.org/shotgunsoftware/python-api)
 [![Build status Windows](https://ci.appveyor.com/api/projects/status/slvw7u4jatvdly98/branch/master?svg=true
 )](https://ci.appveyor.com/project/jfboismenu/python-api/branch/master)
 [![Coverage Status](https://coveralls.io/repos/github/shotgunsoftware/python-api/badge.svg?branch=master)](https://coveralls.io/github/shotgunsoftware/python-api?branch=master)
 
 # Shotgun Python API
 
-Shotgun provides a simple Python-based API for accessing Shotgun and integrating with other tools. This is the official API that is maintained by Shotgun Software (support@shotgunsoftware.com)
+ShotGrid provides a simple Python-based API for accessing ShotGrid and integrating with other tools. This is the official API that is maintained by ShotGrid Software (support@shotgunsoftware.com)
 
 The latest version can always be found at http://github.com/shotgunsoftware/python-api
 
 ## Minimum Requirements
 
-* Shotgun server v2.4.12+.
-* Python v2.6 - v2.7 or v3.7
+* ShotGrid server v2.4.12+.
+* Python v2.7 or v3.7
 
 ## Documentation
-Tutorials and detailed documentation about the Python API are available at http://developer.shotgunsoftware.com/python-api).
+Tutorials and detailed documentation about the Python API are available at http://developer.shotgridsoftware.com/python-api).
 
 Some useful direct links:
 
-* [Installing](http://developer.shotgunsoftware.com/python-api/installation.html)
-* [Tutorials](http://developer.shotgunsoftware.com/python-api/cookbook/tutorials.html)
-* [API Reference](http://developer.shotgunsoftware.com/python-api/reference.html)
-* [Data Types](http://developer.shotgunsoftware.com/python-api/reference.html#data-types)
-* [Filter Syntax](http://developer.shotgunsoftware.com/python-api/reference.html#filter-syntax)
+* [Installing](http://developer.shotgridsoftware.com/python-api/installation.html)
+* [Tutorials](http://developer.shotgridsoftware.com/python-api/cookbook/tutorials.html)
+* [API Reference](http://developer.shotgridsoftware.com/python-api/reference.html)
+* [Data Types](http://developer.shotgridsoftware.com/python-api/reference.html#data-types)
+* [Filter Syntax](http://developer.shotgridsoftware.com/python-api/reference.html#filter-syntax)
```

Comment:
These links don't work right now. Are we sure they are going to be online when this is released?